lua/autorun/webaudio.lua

```diff
@@ -681,6 +681,9 @@ local function isWhitelistedURL(url)
 	if not isstring(url) then return false end
 	url = url:Trim()
 
+	local isWhitelisted = hook.Run("WA_IsWhitelistedURL", url)
```

Comment:
Inline this to just ``if hook.Run("WA_IsWhitelistedURL", url) then return true end``

Reply:
The intention behind doing it with the ~= nil check was to allow returning `true` or `false` from the hook. 
So the addon defining the hook could also return false to explicitly block a url early.

Reply:
That being said, I'm fine changing it if that's not something you want.

Reply:
I doubt multiple addons would be working with this hook, but you can keep the `~= nil` logic I guess